app/crawler/mbc_api.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any
+import aiohttp
+import json
+from datetime import date
+from app.crawler.base import BaseNewsCrawler, Article
+
+
+class MbcNewsApiCrawler(BaseNewsCrawler):
+    async def fetch_articles(self, keyword: str) -> list[Article]:
+        current_id, data_id = await self._get_today_ids()
+        return await self._fetch_news(current_id, data_id, keyword)
+
+    async def _get_today_ids(self) -> tuple[str, str]:
+        cal_url = "https://imnews.imbc.com/news/2025/politics/cal_data.js"
+        
+        async with aiohttp.ClientSession() as session:
+            async with session.get(cal_url) as response:
+                # 해당 파일은 Content-Type이 application/javascript이므로
+                # aiohttp의 response.json()을 사용할 수 없습니다.
+                # 대신 텍스트로 받고 수동으로 json.loads() 처리해야 함
+                raw = await response.text()
+
+        try:
+            # BOM(Byte Order Mark, '\ufeff') 제거 후 양쪽 공백 제거
+            cleaned = raw.lstrip('\ufeff').strip()
```

Comment:
javascript 형식의 문서이기때문에 전처리가 필요한데 간혹가다 자바스크립트 구현체가 포함되어있는 경우도 있고, 배열 마지막 요소 뒤에 콤마(,) 가 붙는 경우가 있습니다.
방금전에 확인해보니 2025년 04월 06일 문서에는 콤마가 포함된 케이스가 있어서 json 파싱이 안되는 문제가 있었습니다.

```suggestion
            cleaned = raw.lstrip('\ufeff').strip()
            cleaned = re.sub(r',(\s*[\]}])', r'\1', cleaned)
```

Reply:
제안해주신 내용은 app/crawler/utils/json_cleaner.py 에 `sanitize_js_style_json()` 를 만들어 해결하였습니다.